setup.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from setuptools import setup
+
+from qiime2 import __version__ as qiime2_version
```

Comment:
my only concern about having this like it is right now is that this package assumes that q2 is already installed in order to install this package. Are we ok given that q2 is not pip installable and the recommended installation procedure is conda? If so, can you add a note on the README stating that this package assumes that Q2 is already installed on the active environment, and add a link to q2's installation instructions?